app/views/coronavirus_form/check_answers.html.erb

```diff
@@ -7,6 +7,8 @@
   <%= render "govuk_publishing_components/components/back_link", { href: previous_path } %>
 <% end %>
 
+<%= @form_responses.inspect %>
```

Comment:
Can you add a note to the commit message about why this was needed and added here?

Reply:
This line shouldn't have been committed.  Have deleted.